requirements.txt

```diff
@@ -13,3 +13,4 @@ jupyter_client
 pandas
 requests
 azure-datalake-store >= 0.0.30
+pytest
```

Comment:
Don't add this here -- it's only needed for tests which is already in `requirements-dev.txt`